docs/configuration.md

```diff
@@ -2296,7 +2296,7 @@ In some cases, you may want to avoid hard-coding certain configurations in a `Sp
 instance, Spark allows you to simply create an empty conf and set spark/spark hadoop properties.
 
 {% highlight scala %}
-val conf = new SparkConf().set("spark.hadoop.abc.def","xyz")
+val conf = new SparkConf().set("spark.hadoop.abc.def", "xyz")
```

Comment:
Likewise you might spot more of these with a crude regex like `","[^), .]`; I found a few more in .scala and .md files

Reply:
I would like to keep this PR minor and only focusing on `.md` files.